assets/js/handlebars/templates/settings-modal-body.handlebars

```diff
@@ -3,7 +3,7 @@
     <label class="switch-button-container">
       <div>
         <span>Night mode</span>
-        <p>Switch to a dark mode version of the website.</p>
+        <p>Use the dark mode version of this website.</p>
```

Comment:
Since this setting is affects other HTML ExDoc docs, we shouldn't imply that it only affects the current website:

```suggestion
        <p>Use the dark mode version for HTML ExDoc documentation</p>
```